src/utils/getPhoneHash.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Reference file:
+ * https://github.com/celo-org/developer-tooling/blob/8572a0f978d1aa01a36775ef4be48c3eafdbb204/packages/sdk/phone-utils/src/getPhoneHash.ts
+ *
+ * The file was removed in later versions of @celo/phone-utils so the function
+ * is taken in its final form before removal.
+ */
+
+import { getIdentifierHash, getPrefixedIdentifier, IdentifierPrefix } from 'src/utils/identifier'
+import { soliditySha3 } from 'web3-utils'
+
+/**
+ * Rerefence implementation uses "web3-utils" version 1.3.6:
+ * https://github.com/celo-org/developer-tooling/blob/8572a0f978d1aa01a36775ef4be48c3eafdbb204/packages/sdk/base/package.json#L30
+ *
+ * In that version the function "soliditySha3" returns "string | null" while the latest version of "web3-utils"
+ * returns "string | undefined". For the sake of compatibility with the original implementation while
+ * also using the latest version of "web3-utils" "?? null" was added to replace possible undefined with null.
+ */
```

Comment:
Nothing to do in this PR, but it looks like sha3 could be replaced with viem's keccak256.

https://github.com/web3/web3.js/blob/9b322052ef266cc57b29080b20053991e0c9a0a9/packages/web3-utils/src/hash.ts#L125

Reply:
Cool, there's a direct replacement with `@noble/hashes/sha3`
And `@noble/hashes` is already used by viem.